src/KubeLibrary/KubeLibrary.py

```diff
@@ -3,6 +3,8 @@
 import ssl
 import urllib3
 from kubernetes import client, config
+import requests
```

Comment:
requests is not needed, see the comment below.

Reply:
request module removed.